src/HTSPConnection.cpp

```diff
@@ -513,7 +514,43 @@ bool CHTSPConnection::SendAuth
   /* Send and Wait */
   msg = SendAndWait0("authenticate", msg);
 
-  return (msg != NULL);
+  // <= htsp v25, msg present = no access, msg missing = access
+  // >= htsp v26, msg should be present in any case
+  if (m_htspVersion <= 25) //
+    access = msg ? 0 : 1;
+  else
+  {
+    if (msg == NULL || (!htsmsg_get_u32(msg, "noaccess", &u32) && u32))
+    {
+      access = 0;
+      Logger::Log(LogLevel::LEVEL_DEBUG, "authentication failed");
+    }
+    else
+    {
+      access = 1;
+      Logger::Log(LogLevel::LEVEL_DEBUG, "authentication successful");
+
+      /* Log received permissions */
+      if (!htsmsg_get_u32(msg, "admin", &u32))
+        Logger::Log(LogLevel::LEVEL_INFO, "admin permissions: %i", u32);
```

Comment:
Can you log these lines with two spaces extra indentation, like we do for demuxer streams? I find it makes the log more readable.